packages/bot-web-ui/src/pages/server-side-bot/add-bot/form-wrappers/useQsSubmitHandler.tsx

```diff
@@ -20,7 +20,7 @@ const useQsSubmitHandler = () => {
     const handleSubmit = async () => {
         const loss_amount = Number(values?.loss ?? 0);
         const profit_threshold = Number(values?.profit ?? 0);
-        const stored_dont_show_warning_value = localStorage?.getItem('qs-dont-show-loss-threshold-warning');
+        const stored_dont_show_warning_value = localStorage?.getItem('SERVER_BOT_LOSS_THRESHOLD_WARNING');
```

Comment:
@sandeep-deriv Bhai since `SERVER_BOT_LOSS_THRESHOLD_WARNING` is being used multiple times then should we create a constant for it and use it everywhere? This may help maintain it better moving forward. 🙏

Reply:
sure 👍